src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,40 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import java.util.List;
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
 public class Main {
+
+    private static final Injector injector = Injector.getInstance("mate.academy");
+    private static final long LAST_BOOK = 3L;
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+
+        Book book = new Book();
+        book.setTitle("Thinking in Java");
+        book.setPrice(BigDecimal.valueOf(19.99));
+        bookDao.create(book);
+        System.out.println(book);
+
+        book = bookDao.findById(LAST_BOOK).orElse(null);
+        System.out.println(book);
+
+        List<Book> books = bookDao.findAll();
+        System.out.println(books);
+
+        book.setTitle("Thinking updated");
+        bookDao.update(book);
```

Comment:
Ensure that the book object is not null before attempting to update it. If the book with LAST_BOOK ID does not exist, this will lead to a NullPointerException.

Reply:
It's test. If we have null, then we have not working project